network/hostport_test.go

```diff
@@ -213,25 +213,25 @@ func (*HostPortSuite) TestParseHostPortsErrors(c *gc.C) {
 		err   string
 	}{{
 		input: "",
-		err:   `cannot parse "" as address:port: missing port in address`,
+		err:   `cannot parse "" as address:port: .*missing port in address.*`,
 	}, {
 		input: " ",
-		err:   `cannot parse " " as address:port: missing port in address  `,
+		err:   `cannot parse " " as address:port: .*missing port in address.*`,
 	}, {
 		input: ":",
 		err:   `cannot parse ":" port: strconv.(ParseInt|Atoi): parsing "": invalid syntax`,
 	}, {
 		input: "host",
-		err:   `cannot parse "host" as address:port: missing port in address host`,
+		err:   `cannot parse "host" as address:port: .*missing port in address.*`,
 	}, {
 		input: "host:port",
 		err:   `cannot parse "host:port" port: strconv.(ParseInt|Atoi): parsing "port": invalid syntax`,
 	}, {
 		input: "::1",
-		err:   `cannot parse "::1" as address:port: too many colons in address ::1`,
+		err:   `cannot parse "::1" as address:port: .*too many colons in address.*`,
```

Comment:
The error changed from:
`cannot parse "::1" as address:port: too many colons in address ::1`
to
`cannot parse "::1" as address:port: address ::1: too many colons in address`